db/production.sql

```diff
@@ -8,8 +8,8 @@ SET client_min_messages = warning;
 SET search_path = public, pg_catalog;
 
 COPY quotes (id, text, author, created_at, updated_at) FROM stdin;
-1	People who think they know everything are a great annoyance to those of us who do.	Isaac Asimov	\N	\N
-2	As a child my family's menu consisted of two choices: take it or leave it.	Buddy Hackett	\N	\N
+1	People who think they know everything are a great annoyance to those of us who do.	Isaac Asimov	2014-07-12 03:40:54.405448	2014-07-12 03:40:54.405448
+2	As a child my family's menu consisted of two choices: take it or leave it.	Buddy Hackett	2014-07-12 03:40:54.405448	2014-07-12 03:40:54.405448
```

Comment:
Nice catch -- these should definitely not have been null. Can you add a validator to the quote model that would reject null entries in the database? 

http://guides.rubyonrails.org/active_record_validations.html